tap.js

```diff
@@ -37,8 +59,9 @@
             this.el.addEventListener('touchend', this, false);
             this.el.addEventListener('touchcancel', this, false);
 
-        } else if (e.type === 'mousedown') {
+        } else if (e.type === 'mousedown' && this.whichbutton(e) === 1) {
```

Comment:
Also I don't think we need to handle the other buttons, as we're only concerned about left click.